### DISCUSSION TITLE
Cisco supplied Vagrant box for N9K

### DISCUSSION THREAD

**Discussion by Kircheneer** [ipspace/netlab#13] [Q&A] [answered]
Any reason as to why you chose not to use the Vagrant box available for download directly rather than rolling your own with the steps linked in the documentation? I was able to get that to work rather easily (in contrast to the expect script).

**Comment by ipspace** [comment, marked as answer]
No idea. Maybe it wasn't available at that time? Please add a note to the documentation (just add it to some other PR) that there's a Vagrant for vagrant-libvirt, and I'll massage it a bit if needed.

**Comment by ipspace** [comment]
Looks like Cisco stopped providing Vagrant boxes in release 10.3(3), and boxes in release 10.2(5) have "limited features"
